.github/workflows/publish-command.yml

```diff
@@ -177,6 +184,58 @@ jobs:
           SLACK_COLOR: DC143C
           SLACK_TITLE: "Failed to publish connector ${{ github.event.inputs.connector }} from branch ${{ github.ref	 }}"
           SLACK_FOOTER: ""
+      - name: Check if connector in definitions yaml
+        run: |
+          connector="airbyte/${{ env.IMAGE_NAME }}"
+          definitionpath=./airbyte-config/init/src/main/resources/seed/
+          sourcecheck=$(yq e ".. | select(has(\"dockerRepository\")) | select(.dockerRepository == \"$connector\")" "$definitionpath"source_definitions.yaml)
+          destcheck=$(yq e ".. | select(has(\"dockerRepository\")) | select(.dockerRepository == \"$connector\")" "$definitionpath"destination_definitions.yaml)
+          if [[ (-z "$sourcecheck" && -z "$destcheck") ]]
+          then exit 1
+          fi
```

Comment:
What do you think a creating a dedicated script for this rather than having it inline?

Reply:
if this script would have inherent value on its own then yes agree it would make sense...

in this case though, we're just failing if we don't find the connector in definitions, and no-op succeeding if we do. I don't think this is particularly valuable on its own.

Reply:
+1 for a script as that is this command is pretty difficult to parse